### ISSUE TITLE
Update documentation for building on android

### ISSUE THREAD

**Issue by ccpp** [FreeRDP/FreeRDP#6932]
In the last two versions of aFreeRDP published on F-Droid,
* Version 2.3.1: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8475
* Version 2.3.2: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8592

a very complicated script is used to build aFreeRDP.
However, since https://github.com/FreeRDP/FreeRDP/pull/6580 building is much easier, it involves only one bash line:
```
../../../scripts/android-build-freerdp.sh \
  --ndk $$NDK$$ \
  --sdk $$SDK$$ \
  --openh264-ndk $$NDK$$/../*r15c \
  --conf android-build-release.conf
```

This should be documented somewhere in FreeRDP (at least in the [FreeRDP's Wiki page on Compilation](https://github.com/FreeRDP/FreeRDP/wiki/Compilation) but maybe also in the repository), and it should be used in future releases on F-Droid.

**Comment by akallabeth** [comment]
Don´t see where the f-droid build can benefit here (the script is mostly setting up `sdk` components and ensuring the most recent OpenSSL tag is used), but :+1:  for documenting these changes.

**Comment by ccpp** [comment]
@akallabeth Regarding F-Droid: You're right about the sdk components and OpenSSL tag in the `prebuild` section, those are still necessary. However, The entire `build` section can be replaced by this single line.

**Comment by akallabeth** [comment]
@ccpp ah, yes, using the new `scripts/android-build-release.conf` can replace the split build, yes.

**Comment by ccpp** [comment]
Yes, but also there is no need to switch directories (popd...pushd) and setting ndk paths redundantly in environment variables any more!
